projects/angular/components/ui-grid/src/components/ui-grid-toggle-columns/ui-grid-toggle-columns.component.scss

```
@@ -1,33 +1,54 @@
 $toggle-col-width: 10em;
 
 .ui-grid-toggle-columns {
-    margin-right: 6px;
-    padding-left: 0;
-
-    mat-select {
-        overflow: hidden;
-        width: 0;
-        // adjustments in order to match trigger icon with optgroup-label icon
-        position: relative;
-        top: -10px;
-        left: -4px;
-    }
+  margin-right: 6px;
+  padding-left: 0;
+
+  mat-select {
+    overflow: hidden;
+    width: 0;
+    // adjustments in order to match trigger icon with optgroup-label icon
+    position: relative;
+    top: -10px;
+    left: -4px;
+  }
+}
+
+.alternatie-mat-select {
```

Careful with the naming and style leaks. Let's encapsulate this in `.ui-grid-toggle-columns`, try not to use `!important`
```suggestion
.use-alternate {
```